templates/shared/base.html

```diff
@@ -1,11 +1,18 @@
+{% macro fmturl(path) -%}
+{% if base_url is defined and base_url -%}
```

Comment:
`var is defined and var` is redundant. Can just do `if base_url`

Reply:
Yes you're right, I'm not used to the field init shorthand and default to being explicit.